Presentation.Web/app/models/ViewModel/Generic/Select2OptionViewModel.ts

```diff
@@ -20,6 +20,7 @@
     }
 
     export interface Select2OptionViewModelWithIndentation<T> extends ISelect2Model {
+        uuid?: string;
```

Comment:
why not use the optionalExtracObject in stead of adding a very specific property????

the purpose of `optionalExtracObject` is to allow you to put anything in there you need such as the original object or a uuid